sphinx_gherkindoc/writer.py

```diff
@@ -317,18 +315,14 @@ def steps(steps: List[behave.model.Step], step_format: str = "{}") -> None:
                 text(step.text)
 
     def examples(
-        example_source: Union[
-            behave_models.Scenario,
-            pytest_bdd_models.Scenario,
-            pytest_bdd_models.Feature,
-        ],
+        example_source: ExampleClass,
         example_source_parent: Optional[behave.model.Feature] = None,
     ) -> None:
         tag_sources = [example_source]
         if example_source_parent:
             tag_sources.append(example_source_parent)
         for example in getattr(example_source, "examples", []):
-            if isinstance(example_source, pytest_bdd_models.Feature):
+            if example_source.__class__.__name__ == "Feature":
```

Comment:
Will this cause a behave feature to take this path now, where as before it wouldn't?

From what I can tell, the behave feature model class name is also "Feature".

Reply:
There is a `behave.Feature` [class](https://github.com/jolly-good-toolbelt/sphinx_gherkindoc/blob/master/sphinx_gherkindoc/parsers/behave.py#L47), but it always returns an empty list, so the outer `for` loop here will never have anything to iterate over, thus the `if` conditional will never arise.

Reply:
@bradsbrown  clarified that as a behave feature will never have examples, this code block will never run.